playbooks/roles/stackdriver/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: Add Stackdriver public key
+  apt_key: url={{ stackdriver_pubkey_url }} state=present
+  tags: ['stackdriver', 'stackdriver:install']
+
+- name: Add Stackdriver repository
+  apt_repository: >
+    repo="deb {{ stackdriver_repo_url }} {{ stackdriver_distro_codename }} main"
+    update_cache=yes
+  tags: ['stackdriver', 'stackdriver:install']
+
+- name: Install Stackdriver agent
+  apt: name=stackdriver-agent
```

Comment:
Hmm, I don't have a strong preference either way. This isn't a mission-critical component, so upgrading it _shouldn't_ break anything. Here are it's dependencies:

```
$ apt-cache depends stackdriver-agent
stackdriver-agent
 |Depends: debconf
  Depends: <debconf-2.0>
    cdebconf
    debconf
  Depends: curl
  Depends: libcurl3
  Depends: libltdl7
  Depends: libyajl2
  Suggests: libmysqlclient18
  Suggests: libpq5
  Suggests: <libhiredis0.10>
  Suggests: default-jre
  Recommends: stackdriver-extractor```